ui/src/components/VolumeListTable.js

```diff
@@ -324,25 +324,7 @@ function Table({
         </div>
         <Body {...getTableBodyProps()}>
           {data.length === 0 ? (
-            <div
-              style={{
-                width: '100%',
-                paddingTop: padding.base,
-                height: '60px',
-              }}
-              className="tr"
-            >
-              <div
-                style={{
-                  textAlign: 'center',
-                  background: theme.brand.primary,
-                  border: 'none',
-                }}
-                className="td"
-              >
-                {intl.translate('no_volume_found')}
-              </div>
-            </div>
+            <EmptyTable>{intl.translate('no_volume_found')}</EmptyTable>
           ) : null}
           {/* <AutoSizer> is a <div/> so it breaks the table layout, 
           we need to use <div/> for all the parts of table(thead, tbody, tr, td...) and retrieve the defaullt styles by className. */}
```

Comment:
I indeed got a warning from the console. I probably need to change the `< EmptyTable/>` to a div. 

```
index.js:1 Warning: validateDOMNesting(...): <tr> cannot appear as a child of <div>.
    in tr (created by styled.tr)
```